docs/guides/modules/storage.rst

```diff
@@ -49,78 +53,88 @@ Change the theme at startup
 API
 ---
 
-.. object:: local_storage
+.. class:: StorageWrapper()
+
+   both :attr:`indexed_db` and :attr:`local_storage` are instances of a dictionary like :class:`StorageWrapper` class.
+
+   .. describe:: is_supported()
+
+      Check if the storage object is supported. Returns a ``boolean``.
+
+   .. describe:: get_store(name)
 
-   local_storage is a dictionary like object.
+      Get or create a ``storage`` object. e.g. ``todo_store = indexed_db.get_store('todos')``. This will create a new storeage object inside the browser's ``IndexedDB``.
+      The :attr:`indexed_db` object is equivalent to ``indexed_db.get_store('default')``. To explore this further, open up devtools and find ``IndexedDB`` in the Application tab.
 
-   .. describe:: list(local_storage)
+   .. describe:: list(store)
 
-      Return a list of all the keys used in :attr:`local_storage`.
+      Return a list of all the keys used in the *store*.
 
-   .. describe:: len(local_storage)
+   .. describe:: len(store)
 
-      Return the number of items in :attr:`local_storage`.
+      Return the number of items in *store*.
 
-   .. describe:: local_storage[key]
+   .. describe:: store[key]
 
-      Return the item of :attr:`local_storage` with key *key*.  Raises a :exc:`KeyError` if *key* is
-      not in :attr:`local_storage`. Raises a :exc:`TypeError` if *key* is not a string.
+      Return the item of *store* with key *key*.  Raises a :exc:`KeyError` if *key* is
+      not in *store*. Raises a :exc:`TypeError` if *key* is not a string.
 
-   .. describe:: local_storage[key] = value
+   .. describe:: store[key] = value
 
-      Set ``local_storage[key]`` to *value*. The *value* must be a json compatible object.
+      Set ``store[key]`` to *value*. If the value is not a JSONable data type it may be stored correctly. e.g. a ``datetime`` object.
```

Comment:
line 84 of storage.rst: should "correctly" read "incorrectly" here?

Reply:
yeap